cms/bundles/templates/bundles/wagtailadmin/action_menu/menu_item.html

```diff
@@ -0,0 +1,15 @@
+{% load i18n wagtailadmin_tags %}
+<button
+    type="submit"
+    name="{{ name }}"
+    value="{{ name }}"
+    class="button button-longrunning"
+    data-controller="w-progress{% if use_shortcut %} w-kbd{% endif %}"
+    data-action="w-progress#activate"
+    data-w-progress-active-value="{{ label_progress }}"
+    {% if use_shortcut %} data-w-kbd-key-value="mod+s" data-w-kbd-scope-value="global"{% endif %}
```

Comment:
Super nit-pick: There's a new line before the if block so we can probably drop the whitespace?

```suggestion
    {% if use_shortcut %}data-w-kbd-key-value="mod+s" data-w-kbd-scope-value="global"{% endif %}
```

Reply:
354bde99